src/Microsoft.IdentityModel.Tokens/BaseConfigurationManager.cs

```diff
@@ -50,6 +50,11 @@ public TimeSpan AutomaticRefreshInterval
         /// </summary>
         public static readonly TimeSpan DefaultRefreshInterval = new TimeSpan(0, 0, 5, 0);
 
+        /// <summary>
+        /// 30 seconds is the time interval that must pass for <see cref="RequestRefresh"/> before obtain the first configuration.
+        /// </summary>
+        public TimeSpan BootstrapRefreshInterval { get; } = new TimeSpan(0, 0, 30, 0);
```

Comment:
TimeSpan(0, 0, 30, 0) is 30 min, not 30 seconds.

Reply:
Change to TimeSpan(0,0,30)